draft-ietf-quic-transport.md

```diff
@@ -2228,6 +2228,12 @@ Note:
   control, which are not expected to be relevant for a closed connection.
   Retransmitting the final packet requires less state.
 
+New packets from unverified addresses could be used to create an amplification
+attack (see {{address-validation}}).  To avoid this, endpoints MUST either limit
+transmission of closing frames to validated addresses - which implies retaining
```

Comment:
I'd remove the "- which implies retraining knowledge of validated addresses -" because it makes the sentence a bit awkward and I think it's unnecessary.